unity/Assets/Scripts/manager/LabelManager.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+
+public static class LabelManager
+{
+    public const string DEV = "Development", PG = "Playground", SELECT = "CharacterSelect";
```

Comment:
컨벤션으로 여겨지는 사항들 아니면 축약어 안 쓰는 게 낫긴 합니당

```suggestion
public const string DEV = "Development", PLAYGROUND = "Playground", SELECT_CHARACTER= "CharacterSelect";
```